core/src/main/java/org/apache/iceberg/TableScanContext.java

```diff
@@ -35,7 +35,7 @@
 
 /** Context object with optional arguments for a TableScan. */
 @Value.Immutable
-abstract class TableScanContext {
+public abstract class TableScanContext {
```

Comment:
This change shouldn't be needed, checking out the code locally

Reply:
will fix this